docs/components/concepts/signals.md

```diff
@@ -0,0 +1,19 @@
+---
+id: signals
+title: "Signals"
+description: "Broadcast signals to trigger signal events"
+---
+
+Signals can be broadcasted within Zeebe. These broadcasts are caught by [signal catch events](../modeler/bpmn/signal-events/signal-events.md).
+Once a signal is caught the process instance will respond accordingly.
+
+## Signal subscriptions
+
+Signals work using subscriptions. When a process encounters a signal catch event it will create a new signal subscription.
+Ths process instance will wait until a signal with a matching name is broadcasted in Zeebe. You can define the signal name
+in the process definition.
```

Comment:
> While this is correct, we don't yet support this behavior. Currently, only the top-level signal start event is supported, which behaves differently. A signal subscription is opened when a process is deployed with a top-level signal start event.

I'm not sure I agree that this is a problem. We document the concept of Signals on this page. We shouldn't care about the elements and if they are support or not. That's why we have these different pages.

> We should also document what happens when a new process version is deployed (subscription is closed).

This is specific to the top-level start event. That's why it's documented on that page, instead of on this one. As I said above I think the concept of signals is separate from the events and what you mention here is in my opinion an event detail, not a concept detail.

We also don't document these things in the concept page of messages.